[2023-04-23T08:46:28.628+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_extraction_dag.task_athletes manual__2023-04-23T08:23:43.182997+00:00 [queued]>
[2023-04-23T08:46:28.640+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_extraction_dag.task_athletes manual__2023-04-23T08:23:43.182997+00:00 [queued]>
[2023-04-23T08:46:28.641+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-23T08:46:28.642+0000] {taskinstance.py:1289} INFO - Starting attempt 3 of 3
[2023-04-23T08:46:28.643+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-23T08:46:28.664+0000] {taskinstance.py:1309} INFO - Executing <Task(_PythonDecoratedOperator): task_athletes> on 2023-04-23 08:23:43.182997+00:00
[2023-04-23T08:46:28.690+0000] {standard_task_runner.py:55} INFO - Started process 5677 to run task
[2023-04-23T08:46:28.695+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_extraction_dag', 'task_athletes', 'manual__2023-04-23T08:23:43.182997+00:00', '--job-id', '1024', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpavf04wtk']
[2023-04-23T08:46:28.699+0000] {standard_task_runner.py:83} INFO - Job 1024: Subtask task_athletes
[2023-04-23T08:46:28.787+0000] {task_command.py:389} INFO - Running <TaskInstance: data_extraction_dag.task_athletes manual__2023-04-23T08:23:43.182997+00:00 [running]> on host 235fa867d994
[2023-04-23T08:46:28.942+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_extraction_dag
AIRFLOW_CTX_TASK_ID=task_athletes
AIRFLOW_CTX_EXECUTION_DATE=2023-04-23T08:23:43.182997+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-23T08:23:43.182997+00:00
[2023-04-23T08:46:28.945+0000] {logging_mixin.py:137} INFO - Fetching for athlete details started
[2023-04-23T08:49:22.568+0000] {local_task_job.py:276} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2023-04-23T08:49:22.699+0000] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 5677. PIDs of all processes in the group: [5677]
[2023-04-23T08:49:22.702+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 5677
[2023-04-23T08:49:22.718+0000] {taskinstance.py:1488} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-04-23T08:49:23.281+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=5677, status='terminated', exitcode=0, started='08:46:27') (5677) terminated with exit code 0
